ocrd/ocrd/processor/helpers.py

```diff
@@ -198,7 +199,10 @@ def run_cli(
     args = [executable, '--working-dir', workspace.directory]
     args += ['--mets', mets_url]
     if log_level:
-        args += ['--log-level', log_level]
+        if isinstance(log_level, str):
+            args += ['--log-level', log_level]
+        else:
+            args += ['--log-level', getLevelName(log_level)]
```

Comment:
```suggestion
        args += ['--log-level', log_level if isinstance(log_level, str) else getLevelName(log_level)]
```

Reply:
We could also change getLevelName to handle that, so we don't need the `isinstance` check here.